addons/mail/static/src/models/thread_view/thread_view.js

```diff
@@ -287,6 +306,13 @@ function factory(dependencies) {
         stringifiedDomain: attr({
             related: 'threadViewer.stringifiedDomain',
         }),
+        /**
+         * Keyboard shortcuts from text input to send message.
```

Comment:
Determines the keyboard shortcuts that are available to send a message from the composer of this thread viewer.